frontend/src/dataUtilities.ts

```diff
@@ -427,3 +459,22 @@ export const transformAccountsData = (data: AccountsGetResponse) => {
     return obj;
   });
 };
+
+interface PaymentData {
+  error: null;
```

Comment:
`error` is unused right? no need to declare it in the interface then?

Reply:
okay. got it. done.